ruby_basic/nikolay.braslavets/home_task_1/6.rb

```diff
@@ -0,0 +1,36 @@
+#For mentors: Not sure about the correct implementation, but it works..
+
+num1 = ARGV[0]
+num2 = ARGV[1]
+operation = ARGV[2]
+
+class Calculator
+  attr_accessor :num1, :num2, :operation
+
+  def initialize()
+    @operation = 'sum'
+    @num1 = 5
+    @num2 = 6
+  end
+
+  def calculate()
+    case @operation
+    when 'sum'
+        puts @num1 + @num2
+      when 'sub'
```

Comment:
Indent when on one level with case
https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Layout/CaseIndentation

Reply:
This is an accident, fix.